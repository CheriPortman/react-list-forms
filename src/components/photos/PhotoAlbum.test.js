import React from 'react';
import { shallow } from 'enzyme';
import PhotoAlbum from './PhotoAlbum';

describe('Photo Album Componenet', () => {
  it('renders a photo album!', () => {
    const photos = [
      '/fake',
      '/fayke'
    ];

    const wrapper = shallow(<PhotoAlbum title="My Photos" photos={photos} />);
    expect(wrapper).toMatchSnapshot();
  });
});
