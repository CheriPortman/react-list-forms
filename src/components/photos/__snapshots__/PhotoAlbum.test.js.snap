// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photo Album Componenet renders a photo album! 1`] = `
<section
  className="PhotoAlbum"
>
  <h2>
    My Photos
  </h2>
  <Photos
    photos={
      Array [
        "/fake",
        "/fayke",
      ]
    }
  />
</section>
`;
